exercises/largest-series-product/.meta/src/reference/java/LargestSeriesProductCalculator.java

```diff
@@ -32,15 +32,15 @@ public long calculateLargestProductForSeriesLength(final int seriesLength) throw
         }
     }
 
-    private void validateStringToSearch() throws IllegalArgumentException {
```

Comment:
I think that these `private` methods can stay private. Do you agree? :slightly_smiling_face: